src/App.js

```diff
@@ -62,11 +67,19 @@ class App extends Component {
     };
   };  // End constructor()
 
+  /** Set the language of the app
+   * @param {object} evnt - An event object, which is not actually used in the function but is passed in by Semantic UI React input components
+   * @param {object} inputProps - An object representing the properties of the Semantic UI React input component which triggered the language change
+  */
   setLanguage = (evnt, inputProps) => {
     var snippets = getTextForLanguage(inputProps.value);
     this.setState({ language: inputProps.value, snippets: snippets });
   };
 
+  /** Set the value of a specified key in the app state's devProps
```

Comment:
Also, currently these keys should only be set to bools (which maybe should be enforced sometime) and will be added as classes to the app's main element if they're value is `true`. This might need to change sometime to allow more flexibility with the dev hud, but I don't currently have a good picture in my head of implementation...

Reply:
So, people should correct me if I'm wrong, but I think that by having the "`{boolean}`" in `@param {boolean} value - The value to be set...`, we might already be expressing in the way JSDoc intends that the value should only be a boolean. But perhaps this bears repeating in the description too. I think the part about the keys being added as classes certainly seems like a good idea to include. (I've done so in 65005253efce1890daeeb30c38d74c8ea9b20a16)

Reply:
For the record, I do think that's a great point about the bools already having a type. I sometimes forget how the documentation works :P